Documentation/CRDs/Cluster/external-cluster/topology-for-external-mode.md

```diff
@@ -0,0 +1,201 @@
+# Topology provisioning for external mode
+
+## Background
+For Block storage (RWO), Enable Topology awareness for PV provisioning. Preferably, picking the replica 1 Storage Class for this AZ or creating a volume in a relevant AZ using a single storage class. It will help in providing Isolation to application/pods in AZ
+
+## Usage
+Applications like Kafka will have a deployment with number of instances. When using more than one instances, each service instances will create a new claim and will be located in a different zone. The configuration will include PVC and storage class. We will need to use replica 1 storage class and we will have PVC equivalent to instances, each PV is expected in a different zone.
```

Comment:
```suggestion
Applications like Kafka will have a deployment with multiple running instances. Each service instance will create a new claim and is expected to be located in a different zone. Since the application has its own redundant instances, there is no requirement for redundancy at the data layer. A storage class is created that will provision storage from replica 1 Ceph pools that are located in each of the separate zones.
```